connectivity/netsocket/source/TCPSocket.cpp

```diff
@@ -173,6 +173,16 @@ nsapi_size_or_error_t TCPSocket::sendto(const SocketAddress &address, const void
     return send(data, size);
 }
 
+nsapi_size_or_error_t TCPSocket::sendto_control(const SocketAddress &address, const void *data, nsapi_size_t size,
+                                                nsapi_msghdr_t *control, nsapi_size_t control_size)
+{
+    // FIXME: Implement
```

Comment:
Can you rather create an tracking issue for this and remove the comment (reference the issue in the commit message, it can be easily found then)

Reply:
Done, new issue - https://github.com/ARMmbed/mbed-os/issues/15057

Reply:
Issue number should be added in the amended commit message when merging this PR into master